fuse/dl/models/backbones/backbone_mvit.py

```diff
@@ -0,0 +1,168 @@
+"""
+(C) Copyright 2021 IBM Corp.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Created on June 30, 2021
+
+"""
+
+import torch.nn as nn
+from torchvision.models.video.mvit import MViT, mvit_v2_s, mvit_v1_b, MViT_V2_S_Weights, MViT_V1_B_Weights
+from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple
+from dataclasses import dataclass
+from fuse.dl.models.heads import Head3D
+from torch import Tensor
+import torch
+
+
+@dataclass
+class MSBlockConfig:
+    num_heads: int
+    input_channels: int
+    output_channels: int
+    kernel_q: List[int]
+    kernel_kv: List[int]
+    stride_q: List[int]
+    stride_kv: List[int]
+
+
+class BackboneMViT(nn.Module):
+    def __init__(
+        self,
+        in_channels: int = 3,  # can be 1
+        input_dim: List[int] = [16, 224, 224],
+        pretrained: bool = False,
+    ) -> None:
+        """
```

Comment:
Please add details - two sentences about mvit + link to pytorch or original paper.